bindings/py/tests/algorithms/spatial_pooler_test.py

```diff
@@ -177,6 +177,19 @@ def testNupicSpatialPoolerPickling(self):
     self.assertEqual(sp.getNumColumns(), sp2.getNumColumns(),
                      "Simple NuPIC SpatialPooler pickle/unpickle failed.")
 
+  def testNupicSpatialPoolerSavingToString(self):
+    """Test writing to and reading from NuPIC SpatialPooler."""
+
+    # Simple test: make sure that writing/reading works...
+    sp = SP()
+    s = sp.writeToString()
+
+    sp2 = sp.readFromString(s)
```

Comment:
there is not `loadFromString`

Reply:
Oops. Got careless. Well in my defense there wasn't the unit test could get far enough to check it. ;-)